src/javascript/app/pages/trade/charts/highchart.js

```diff
@@ -608,12 +608,14 @@ const Highchart = (() => {
                     i++;
                 }
                 const last = parseInt(last_data.x || last_data[0]);
-                if (last > (end_time * 1000) || last > ((sell_time || sell_spot_time) * 1000)) {
+                if (last >= (end_time * 1000) || last >= ((sell_time || sell_spot_time) * 1000)) {
```

Comment:
I think it was `>` so that it would draw one tick after the end, is there any reason to remove that?

Reply:
oh it was intentional? i will revert it then 👍